### PR TITLE
[SEMI-MODULAR] [READY?] Antagonist Tips - preference for it included.

### PR DESCRIPTION
**Pull request by SgtHunk**
## About The Pull Request

Don't mind me just gonna see if this works in a TM
Also DNM until I make a readme and see if it works in an enviroment that isn't local

## Why It's Good For The Game

Blah blah blah antag tip good and its a preference :)

## Changelog
:cl:
qol: Antagonist tips - as a preference.
/:cl:

### CODE REVIEW

File: code/___fulp_defines/fulp_defines.dm [hunk 1]
```diff
@@ -61,3 +61,25 @@
...
+ *	Antag Tip Defines
+ */
+#define TRAITOR_TIPS "traitor"
+#define IAA_TIPS "iaa"
+#define IAA_AI_TIPS "iaa_ai"
+#define MALF_TIPS "malf_ai"
+#define NUKIE_TIPS "nukie"
+#define WIZARD_TIPS "wizard"
+#define WIZARD_APPRENTICE_TIPS "wiz_apprentice"
+#define IMPOSTER_TIPS "imposter"
```
Comment (JohnFulpWillard) [inline code/___fulp_defines/fulp_defines.dm:75]: .

**Comment by JohnFulpWillard** [comment]
bobux no tgui

**Comment by remove32** [comment]
what in the goddamn

**Comment by remove32** [comment]
isnt this basically the same thing as job policy stuff?

**Comment by SgtHunk** [comment]
> isnt this basically the same thing as job policy stuff? 

This is for antagonists, but only mildly related.
Job policy is for what the job should/shouldnt do, what is expected of them.
This is meant to give antagonists basic information of their role and general directions.

**Comment by remove32** [comment]
sure but what i mean is could you feasibly just use the way that we do job policy, but also for antags?

popping up windows and shit will be annoying and a lot of overhead, might be difficult. 
could use the notes feature? not many people use it though...
just some ideas

**Comment by SgtHunk** [comment]
Not many people read chat, apparently.
This has been thoroughly witnessed by @JohnFulpWillard who has needed to add A LOT of warning texts regarding mechanics for Bloodsuckers and yet people still fail to notice them.

Pop-ups are more effective. And well, if someone doesn't like it they can disable the tips on their preferences.

**Comment by JohnFulpWillard** [comment]
> This has been thoroughly witnessed by @JohnFulpWillard who has needed to add A LOT of warning texts regarding mechanics for Bloodsuckers and yet people still fail to notice them.

I have made the persuasion rack borderline flood the chat telling them how to use it and I still get tons of questions asking how to wrench it in place

**Comment by remove32** [comment]
> 
> 
> Not many people read chat, apparently.

aint that the fuckin truth

that being the case maybe we should do the reverse and roll the job tips/guidelines into this. im fine starting here though, we can work up to that. just thoughts.

has this been TM'd yet? we can try it out on the test server when i get off work

**Comment by SgtHunk** [comment]
It has been TM'd on the test server. Since I had already tested, it did work... for the most part.
And same as last TM, the HTML pop-ups were empty.

I had a theory that maybe a full merge would fix that (as it works on local yet doesn't work on testmerge), and would like to test it after I get rid of unused images, properly make a readme file and slightly tidy it up. I can make a revert PR the second it gets merged in case it breaks anything, if it is any concern.

**Comment by Helianthus00** [comment]
I'll merge this tonight or tomorrow so we can see if it works and revert if needed.

**Comment by SgtHunk** [comment]
> I'll merge this tonight or tomorrow so we can see if it works and revert if needed.

Tested this alongside wind on the test server, worked. Issue was related to not clearing up cache iirc.

**Pull request merged**